CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
 # CHANGELOG
+
+## 3.7.0
+#### New Functionality
+
+- MarkLogic 12 users can now use the `base64Encode` and `base64Decode` methods compatible with server vector functions.
```

Comment:
Identify the location of these functions (and use "function" instead of "method" as "function" is more appropriate for JS code) - `/lib/vector-util.js` - as there are also the `vec` encode functions which aren't what's being added here. 

And technically, there's no dependency on the server here - these functions can be used regardless of the server version. I think the better way to describe it is (this also explains the "why" of why someone would care about these new functions):

"Can now encode and decode vectors through the `base64Encode` and `base64Decode` functions in the new `lib/vector-util.js` library. Encoding vectors before writing them to documents in MarkLogic 12 helps reduce the amount of disk space and memory consumed by vectors."